atlas/AtlasMl/atlasml/dependencies.py

```
@@ -23,10 +24,29 @@ def _get_api_key(request: Request) -> str:
     return authorization_header
 
 
+def get_api_keys() -> List[APIKeyConfig]:
+    """Dependency to get API keys from settings."""
+    return get_settings().api_keys
+
+
 class TokenValidator:
+    def __init__(self, api_keys: List[APIKeyConfig] = Depends(get_api_keys)):
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)